umbraco/Umbraco-CMS
# languageCultureName null makes umbraco explode

# Notes
- Fixes an issue where if a language had their languageCultureName as null in the database, it would break.

# How to test
- Run umbraco
- Create a language
- Go to the `umbracoLanguage` table in the database
- set `languageCultureName` to null 
![image](https://user-images.githubusercontent.com/70372949/176690956-5f21791b-6604-4462-a3b3-e191383ff211.png)
- Go to the language in the backoffice, this should no longer explode
